### DISCUSSION TITLE
Daylight distribution

### DISCUSSION THREAD

**Discussion by bcholewa** [protomaps/basemaps#83] [Q&A]
Hello, we are looking into using https://daylightmap.org/ with building footprints. So far our attempts to get this working failed.
Anybody tried to get from daylight .pbf all the way to pmtiles format?

**Comment by do-me** [comment]
Not yet, but would be interested too!

**Comment by wipfli** [comment]
Daylight has been phased out. I think November was the last release. If you want to get building footprints from Overture into tiles, you can do something https://github.com/OvertureMaps/overture-tiles/blob/main/profiles/Buildings.java. This works with Parquet already...

**Reply by bdon** [reply]
we have tentative plans to add an option for Overture as a alternate source for some layers, but right now the simplest way is to use the overture tilesets from https://docs.overturemaps.org/examples/overture-tiles/ and overlay them on a basemap.
